src/main/java/school/lemon/changerequest/java/io/hw2/FileCopyUtilImpl_bufferedStreams.java

```diff
@@ -0,0 +1,28 @@
+package school.lemon.changerequest.java.io.hw2;
+
+import java.io.*;
+import java.nio.file.FileAlreadyExistsException;
+
+/**
+ * Created by lera on 07.02.17.
+ */
+public class FileCopyUtilImpl_bufferedStreams implements FileCopyUtil {
+    @Override
+    public boolean copy(String src, String dst) throws FileNotFoundException, FileAlreadyExistsException, FileCopyException {
+        File fileSource = new File(src);
+        if (!fileSource.exists()) throw new FileNotFoundException(src);
+        File fileDst = new File(dst);
+        if (fileDst.exists()) throw new FileAlreadyExistsException(dst);
+        try (InputStream is = new FileInputStream(src);
```

Comment:
This is not actually buffered implementation we expect.
Try to use `BufferedInputStream` and `BufferedOutputStream`. They could be used in pair with your own buffer.